src/Model/Datasource/DatasourceInterface.php

```diff
@@ -15,5 +15,7 @@
 
 interface DatasourceInterface
 {
+    public const DEFAULT_MAX_PER_PAGE = 100;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Shouldn't it be 10 like written in the description of the PR ? Or this is not the increased batch size that you talked about ?

Reply:
by default, there is no `MaxPerPage` value, so we have batches of 10 products. 
From this PR, I propose to set up batches of 100 products.

I could see about putting the value in the configuration file...